.changeset/hot-mayflies-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
+---
+
+Utilizes `<button>` inside of `ActionList.Item` instead of relying on `<li>` as an interactive item.
```

Comment:
What would be the best way to convey that this feature is feature flagged and not available outside of the flag? 🤔

Reply:
Curious to learn that as well!